yax/core/src/main/scala/doobie/util/fragments.scala

```diff
@@ -0,0 +1,51 @@
+package doobie.util
+
+import doobie.imports._
+
+#+scalaz
+import scalaz._, Scalaz._
+#-scalaz
+#+cats
+import cats.{ Reducible => Foldable1, _}, cats.implicits._
+#-cats
+
+/** Module of `Fragment` constructors. */
+object fragments {
+
+  /** Returns `f IN (fs0, fs1, ...)`. */
+  def in[F[_]: Foldable1, A: Param](f: Fragment, fs: F[A]): Fragment =
```

Comment:
Yeah but that's kind of janky. Should we add a `notIn` constructor? I fear this will never end but at least we can do the obvious ones.